examples/point-cloud-laz/app.js

```diff
@@ -139,9 +139,14 @@ class Example extends PureComponent {
   }
 
   _renderLazPointCloudLayer() {
-    return this.state.points.length && new PointCloudLayer({
+    const {points} = this.state;
+    if (!points || points.length === 0) {
```

Comment:
With the support for falsy values in the layer array that I merged yesterday, you should be able to just do
```
return points && points.length && newPointCloudLayer({
```